ml-agents-envs/mlagents_envs/side_channel/stats_side_channel.py

```diff
@@ -13,6 +15,10 @@ class StatsAggregationMethod(Enum):
     MOST_RECENT = 1
 
 
+StatList = List[Tuple[float, StatsAggregationMethod]]
+EnvironmentStats = Mapping[str, StatList]
```

Comment:
👍

Reply:
I originally had DefaultDict here, but Pycharm was complaining about my unit test that I was passing a dict instead.